deploy/melange.yaml.tmpl

```diff
@@ -19,13 +19,26 @@ environment:
       - go
       - nodejs
       - yarn
+      - make
 
 pipeline:
   - runs: |
       set -x
       export DESTDIR="${{targets.destdir}}"
       mkdir -p "${DESTDIR}"
 
+      # Build Helm from PR 13439
+      # to be removed later when latest Helm supports --take-ownership flag
+      git clone --depth 1 https://github.com/helm/helm.git
+      cd helm
+      git fetch --depth 1 origin pull/13439/head:pr-13439
+      git checkout pr-13439
+      make
+      chmod a+x bin/helm
+      mv bin/helm "${DESTDIR}/usr/local/bin/"
```

Comment:
you need to run `mkdir -p ${DESTDIR}/usr/local/bin/` first. which is why CI is failing